application/src/main/java/org/togetherjava/tjbot/config/OofsAndLmaosConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+@JsonRootName("oofsAndLmaos")
+public final class OofsAndLmaosConfig {
+    private final String oofEmojiName;
+    private final String lmaoEmojiName;
+    private final long starboardChannelId;
+
+    @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
+    public OofsAndLmaosConfig(String oofEmojiName, String lmaoEmojiName, long starboardChannelId) {
```

Comment:
missing all the `@JsonProperty` stuff, see the main config

Reply:
Done